arrayWithRandomValues.java

```diff
@@ -0,0 +1,69 @@
+public class arrayWithRandomValues {
+    public static void main(String[] args){
+        int []array = array();
+        average(array);
+        min(array);
+        max(array);
+        bubble(array);

```

Comment:
Names of methods should show what they exactly do. example for average "getArrayAverage"